operator/cmd/mesh/manifest-apply.go

```diff
@@ -46,8 +49,13 @@ type manifestApplyArgs struct {
 	// set is a string with element format "path=value" where path is an IstioOperator path and the value is a
 	// value to set the node at that path to.
 	set []string
+	// path where to save and store the certificates
+	certDir string
 }
 
+// default dir to store the generated certificate when we run under dryRun mode
```

Comment:
I'm not sure about this - it implies that we must run as root or istio-proxy on VMs. 

How about "./etc/istio" - so at least we can test and use this as regular user ?

Note that K8S uses /var/run/secrets - so my preference would be ./var/run/secrets/istio-certs - 
so we can both test as non-root, and if running in k8s we use the std place. Some tools already 
know how to treat that dir ( like docker builder )

Reply:
Also 'dryRun' typically means we don't store.

Reply:
Agree, Let's do not store when `dryRun`